src/qspdlog_model.cpp

```diff
@@ -145,3 +160,39 @@ QVariant QSpdLogModel::headerData(
 
     return QVariant();
 }
+void QSpdLogModel::setForegroundMapping(
+    const std::string& loggerName, std::optional<QBrush> brush
+)
+{
+    if(brush.has_value())
+        _foregroundMappings[loggerName] = brush.value();
+    else if(_foregroundMappings.contains(loggerName))
+        _foregroundMappings.erase(loggerName);
+}
+std::optional<QBrush> QSpdLogModel::getForegroundMapping(
+    const std::string& loggerName
```

Comment:
`const std::string&` -> `std::string_view`
```suggestion
    std::string_view loggerName
```